test/In.ProjectEKA.HipServiceTest/DataFlow/FakeHttpMessageHandler.cs

```diff
@@ -0,0 +1,22 @@
+namespace In.ProjectEKA.HipServiceTest.DataFlow
+{
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class FakeHttpMessageHandler : DelegatingHandler
+    {
+        private readonly HttpResponseMessage fakeResponse;
+
+        public FakeHttpMessageHandler(HttpResponseMessage responseMessage)
+        {
+            fakeResponse = responseMessage;
+        }
+
+        protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
+            CancellationToken cancellationToken)
+        {
+            return await Task.FromResult(fakeResponse);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)